mountpoint-s3/src/main.rs

```diff
@@ -343,16 +343,28 @@ fn mount(args: CliArgs) -> anyhow::Result<FuseSession> {
         .transpose()
         .context("Failed to parse endpoint URL")?;
 
-    match retrieve_instance_type() {
+    let ec2_instance_type = match retrieve_instance_type() {
         Ok(ec2_instance_type) => {
-            // TODO: config throughtput_target_gbps here.
             tracing::info!("The EC2 instance type is: {:?}", ec2_instance_type);
+            ec2_instance_type
         }
-        Err(e) => tracing::error!("failed to detect EC2 instance type. {:?}", e),
-    }
+        Err(e) => {
+            tracing::error!("Failed to detect EC2 instance type: {:?}", e);
```

Comment:
We probably should use `warning!` instead of `error!` because the mount is not going to fail even if we cannot detect the instance type.